entries/h/homedepot.com.json

```diff
@@ -3,7 +3,9 @@
     "domain": "homedepot.com",
     "url": "https://www.homedepot.com",
     "tfa": [
-      "sms"
+      "sms",
+      "call",
+      "email"
```

Comment:
Can you provide a screenshot of the activation page? Please blur any personal information.

Reply:
Here's a screenshot:

![upload](https://github.com/2factorauth/twofactorauth/assets/93500165/f98c986f-e1de-48d0-9ef4-77ade9cf4313)

(Fun fact: Signal Desktop has some more marker thickness controls for editing photos than Signal Android; not sure how Signal iOS is like.)

Also, I used manual redaction, as the [warning](https://www.privacyguides.org/en/data-redaction/#privacyblur) for PrivacyBlur on Privacy Guides indicates that it's possible to recover text that has been blurred.

So, drawing opaque rectangles (as [implied](https://nitter.1d4.us/Snowden/status/1423018727361060865#m) by Snowden on Twitter) or opaque redaction (such as the old-school way of redacting paper copies of classified documents) is the safer way of redacting text.

Reply:
Is there another page that shows that phone calls and emails act as two-factor authentication?

Reply:
So I had to turn off 2FA in order to see this again, which Home Depot calls "logging in without a password".  It's sort of laconic and I don't think Home Depot has any support page explaining what's going on.

This is what the page looks like before turning on Home Depot's 2FA, which is called "Sign In with a Password":
![upload1](https://github.com/2factorauth/twofactorauth/assets/93500165/981e5b3a-2ddd-44a5-ab6c-71283aaf0ff1)

When you select to activate 2FA, Home Depot calls this "Sign In without a Password":
![upload2](https://github.com/2factorauth/twofactorauth/assets/93500165/91f8b2d3-eeee-43c8-b406-2d8e2a13bfd3)

Although Home Depot requires a cell phone number to create an account, which will be verified with a code in an SMS message (or a SMS capable VOIP number); Home Depot will default to sending the 2FA code for initial setup and all subsequent logins via e-mail:
![upload3](https://github.com/2factorauth/twofactorauth/assets/93500165/946ea6be-da11-4626-b8ba-af4292cc9847)

This is what the page will look like upon successful activation of 2FA:
![upload4](https://github.com/2factorauth/twofactorauth/assets/93500165/554b5b55-25ca-4df2-9183-841bcd02f13b)

Reply:
What their sign in without password describes is not 2FA.
> Instead of entering a password you may sign into your account using a one-time passcode that we will send to you.

What happens if you toggle the 2FA switch?

Reply:
@lazytownfan if a password isn't required then it's still just 1 factor.

Reply:
So, would this not fall under 2FA Directory's definition of 2FA?

Reply:
Exactly